src/smc-util/db-schema/site-defaults.ts

```diff
@@ -6,6 +6,7 @@
 // Default settings to customize a given site, typically a private install of CoCalc.
 
 const { is_valid_email_address } = require("smc-util/misc");
+const { DNS } = require("../theme");
```

Comment:
Change this to be an import (and same for above).  If you use imports then typescript provides potentially useful information via type inference.  If you use require, it disables all of that.  Never ever ever use require unless you have no choice (e.g., coffeescript code).